### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by Movgto**
## Please attention code reviewer reading this 🙏 
I have been having some communication issues with my partner this week, I tried to talk to him and work together but he tells me he's been having some external situation and that's why he's been short in time with the activities, I've been waiting to work with him but he's still delaying the work and there are 3 projects pending to deliver, and I don't think he's going to be able to complete the work, so please can you accept this exercise solo? I already reached out to student success via zendesk about my situation but I haven't got an answer 😞

### CODE REVIEW

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @Movgto 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 

- [ ] This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#work-and-submission-mode). Please always reach out to the student success early enough when you have collaboration issue with your partner(s). 

- [ ] I'm sorry, but I cannot proceed to review your code until you are permitted to work solo by the success team and attaching the screenshot of your claim in your PR 🙂

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by wayungi — COMMENTED** [review]
Hi @Movgto , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project a pair programming project done solo.  Kindly note that when you experience troubles with your partner you need to reach out to student success early enough and attach a screen shot of the their response to your PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/validation.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  if (emailOk && textOkArr.includes(false) === false) {
+    fetch('https://formspree.io/f/mjvdnklq', {
+      method: 'POST',
+      body: new FormData(document.querySelector('form'))
+    });
+
+    alert('Your information has been succesfully delivered!');
+  }
+}
```
Comment (mwanawabangona) [inline js/validation.js]: - [ ] `required`
- [ ] the project requirements states that 
`When the user submits the form, you check if the email is in lower case.
If the validation is OK, the form is sent.
If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.`
However, your project is allowing to submit a None lower case email 😞 
please make amends 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Movgto,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @Movgto , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No linter errors ✔️
✅ Repo looks professional 💼
✅ Good quality commits

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**